public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -609,7 +610,7 @@ export const Explorer = ({
       indexFields: explorerFields,
       userConfigs: {
         ...visualizationSettings,
-        ...processMetricsData(explorerData.schema, visualizationSettings),
+        ...processMetricsData(explorerData.schema),
```

Comment:
Any reason why `visualizationSettings` was removed?

Reply:
`visualizationSettings` wasn't used any where in the `processMetricsData` function and it was redundant

Reply:
Got it, It would be great to just run sanity tests once in event analytics to be sure. Thanks :)

Reply:
sure! I checked every single instance of this function's usage across all files, hope that helps :)